udata/core/dataset/api_fields.py

```diff
@@ -44,6 +44,11 @@
         required=True, enum=RESOURCE_TYPES.keys()),
     'format': fields.String(description='The resource format', required=True),
     'url': fields.String(description='The resource URL', required=True),
+    'permalink': fields.String(description='The permanent URL redirecting to '
```

Comment:
I think we should rename that attr to `latest` because `permalink` is confusing given that it is not a permalink.